src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/OperationsController.cs

```diff
@@ -72,14 +72,27 @@ public IActionResult GetOperationStatus(Guid operationId)
         [HttpDelete("{operationId}")]
         [ProducesWithProblemDetails(ContentTypes.ApplicationJson)]
         [ProducesResponseType(typeof(void), StatusCodes.Status200OK)]
-        public IActionResult CancelOperation(Guid operationId)
+        [ProducesResponseType(typeof(void), StatusCodes.Status202Accepted)]
+        public IActionResult CancelOperation(
+            Guid operationId,
+            [FromQuery]
+            bool gracefulStop = false)
```

Comment:
I would just call this `stop`; we're never going to add an `ungracefulStop`/`abruptStop` parameter.